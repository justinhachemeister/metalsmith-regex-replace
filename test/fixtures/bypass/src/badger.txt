badger badger |badger| badger
|badger| badgerbadger
badger |badger|
|badger|
badger |badger| badger badger |badger| badger