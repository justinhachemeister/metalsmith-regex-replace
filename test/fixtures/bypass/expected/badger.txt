snake snake badger snake
badger badgerbadger
snake badger
badger
snake badger snake snake badger snake